docs/content/v2024.2/explore/ysql-language-features/pg-extensions/extension-pgcron.md

```diff
@@ -21,7 +19,7 @@ If the pg_cron leader node fails, another node is automatically elected as the n
 
 ## Set up pg_cron
 
-pg_cron in YugabyteDB is {{<tags/feature/ea>}}. Before you can use the feature, you must enable it by setting the `enable_pg_cron` flag. To do this, add `enable_pg_cron` to the `allowed_preview_flags_csv` flag and set the `enable_pg_cron` flag to true on all YB-Masters and YB-TServers.
+Before you can use the feature, you must enable it by setting the `enable_pg_cron` flag. To do this, add `enable_pg_cron` to the `allowed_preview_flags_csv` flag and set the `enable_pg_cron` flag to true on all YB-Masters and YB-TServers.
```

Comment:
If no longer ea why does this flag need to be set

Reply:
@hari90 - I believe this section needs updates. We don't require setting `enable_pg_cron` flag. Correct? 
Also, is there any update to the command - 
```sh
./bin/yugabyted start --master_flags "allowed_preview_flags_csv={enable_pg_cron},enable_pg_cron=true" --tserver_flags "allowed_preview_flags_csv={enable_pg_cron},enable_pg_cron=true,ysql_cron_database_name=db1" --ui false
```

Reply:
The flag is still needs and will be even when we call this GA. pg_cron requires an extra backend process to run which we want to avoid if cron is not required by the user.